src/inverseRobot.js

```diff
@@ -25,6 +25,19 @@
  */
 function inverseRobot(robot) {
   // write code here
+
+  // write code here
+  const newRobot = {};
+
+  for (const keys in robot) {
+    newRobot[robot[keys]] = keys;
+  }
+
+  if (Object.keys(robot).length !== Object.keys(newRobot).length) {
+    return null;
+  }
+
```

Comment:
try to rewrite this part of the code with `hasOwnProperty`
![image](https://user-images.githubusercontent.com/53319174/183445957-8eea1178-f3ca-4d9c-bb38-903c376c8d7c.png)